namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-official-visits-dev/resources/elasticache.tf

```diff
@@ -0,0 +1,32 @@
+module "elasticache_redis" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=8.0.0"
+  vpc_name               = var.vpc_name
+  application            = var.application
+  environment_name       = var.environment
+  is_production          = var.is_production
+  business_unit          = var.business_unit
+  infrastructure_support = var.infrastructure_support
+  team_name              = var.team_name
+  number_cache_clusters  = var.number_cache_clusters
+  node_type              = "cache.t4g.small"
+  engine_version         = "7.1"
+  parameter_group_name   = "default.redis7"
+  namespace              = var.namespace
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "elasticache_redis" {
+  metadata {
+    name      = "elasticache-redis"
+    namespace = var.namespace
```

Comment:
Might be good to call it `hmpps-official-visits-ui-elasticache-redis` here? To fit the pattern adopted by other secret names, even though they're all a bit long winded.